src/controllers/userController.js

```diff
@@ -0,0 +1,76 @@
+const userService = require('../services/userService');
+
+const get = async (req, res) => {
+  res.send(await userService.getAllUsers());
+};
+
+const getOne = async (req, res) => {
+  const { id } = req.params;
+
+  const user = await userService.getUserById(id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = async (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = await userService.updateUser({ id, name });
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
```

Comment:
It seems like you're checking the type of the 'name' after trying to update the user. This could lead to problems if the name is not a string. You should move this check before calling the 'updateUser' method.